sway-core/src/control_flow_analysis/analyze_return_paths.rs

```diff
@@ -262,32 +244,6 @@ fn connect_declaration(
             Ok(leaves.to_vec())
         }
         ErrorRecovery => Ok(leaves.to_vec()),
-        Break { .. } => {
-            let entry_node = graph.add_node(node.into());
-            for leaf in leaves {
-                graph.add_edge(*leaf, entry_node, "".into());
-            }
-            match break_to_node {
-                Some(break_to_node) => {
-                    graph.add_edge(entry_node, break_to_node, "".into());
-                    Ok(vec![break_to_node])
-                }
-                None => Err(CompileError::BreakOutsideLoop { span }),
```

Comment:
I suppose this error will be emitted later in `ir_generation` so that's okay. However, are the graph connections above no longer needed? The idea was to transition the control flow to `break_to_node` or `continue_to_node` because everything after a break or continue in a given scope is essentially dead.

Reply:
This PR *shouldn't* change the current behaviour at all since the way `break` and `continue` expressions are handled is (if I made no mistakes) exactly the same as if they were enclosed in a single-declaration `CodeBlock`. I think the problem here is that `break_to_node` and `break_to_continue` weren't getting plumbed through to where they were needed anyway, so those variables ended up becoming unused and so I removed them.